config/initializers/doorkeeper_openid_connect.rb

```diff
@@ -27,12 +27,36 @@
   end
 
   claims do
-    claim :preferred_username, :scope => :openid do |resource_owner, _scopes, _access_token|
+    claim :preferred_username, :response => [:id_token, :user_info] do |resource_owner, _scopes, _access_token|
       resource_owner.display_name
     end
 
     claim :email, :scope => :read_email, :response => [:id_token, :user_info] do |resource_owner, _scopes, _access_token|
       resource_owner.email
     end
+
+    claim :email_verified, :scope => :read_email, :response => [:id_token, :user_info] do |resource_owner, _scopes, _access_token|
```

Comment:
I'm not sure we want to get into the business of claiming to know whether emails are valid, and that flag is I think always true for any active account anyway.